pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -589,6 +589,7 @@ self: super: {
   # Tests attempt to use NPM to install from the network into
   # /homeless-shelter. Disabled.
   purescript = dontCheck super.purescript;
+  purescript_0_10_1 = dontCheck super.purescript_0_10_1;
```

Comment:
Looks like `purescript_0_10_1` has already been replaced with `purescript_0_10_2` in `master`:

```
$ nix-env -qaP -A nixos.haskellPackages '.*purescript.*'
...
nixos.haskellPackages.purescript_0_10_2           purescript-0.10.2
nixos.haskellPackages.purescript                  purescript-0.9.3
...
```